samples/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "samples",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.3",
+    "bootswatch": "^5.2.2",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.6.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
```

Comment:
all these need to be converted to configs are moved to the main package.json

Where did you refer the samples from ?

Reply:
I use [create-react-app](https://create-react-app.dev/) to build this sample folder from zero. You are right, Arun, merge this `package.json` with the main one is better. Let me try it.